scripts/migration.sh

```diff
@@ -34,6 +34,12 @@ printf "Generating migration boilerplate for patch level $NEW_LEVEL..."
 
 echo "SET NAMES utf8mb4 COLLATE utf8mb4_bin;\n" > "$FWD_SCHEMA"
 echo "-- TODO: Implement your forward migration here\n" >> "$FWD_SCHEMA"
+
+echo "SELECT @patchlevel := value FROM `dbMetadata` WHERE `name` = `schema-patch-level`;\n">> "$FWD_SCHEMA"
+echo "IF @patchlevel != $PREV_LEVEL THEN\n">> "$FWD_SCHEMA"
+echo "SIGNAL SQLSTATE '45000' SET MYSQL_ERRNO = 1643, MESSAGE_TEXT = 'Could not migrate to next patch';\n">> "$FWD_SCHEMA"
```

Comment:
Let's also add a couple of spaces before `SIGNAL SQLSTATE` so that this line is sensible indented in the migration. And we still need to come up with a more helpful `MESSAGE_TEXT` of course.